editor.html

```diff
@@ -373,6 +373,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                                 <li><span title="Spanish"><a lang="es" href="#" class="action lang-choice" id="es" tabindex="122">Español</a></span></li>
                                 <li><span title="Polish"><a lang="pl" href="#" class="action lang-choice" id="pl" tabindex="123">Polski</a></span></li>
                                 <li><span title="Croatian"><a lang="hr" href="#" class="action lang-choice" id="hr" tabindex="124">Hrvatski</a></span></li>
+                                <li><span title="Chinese (simplified)"><a lang="hk" href="#" class="action lang-choice" id="hk" tabindex="125">简体中文</a></span></li>
                             </ul>
                             <div><a id="add-language-link" class="action" title="Add a language translation to the Editor" href="https://support.microbit.org/support/solutions/articles/19000106022-translating-the-micro-bit-python-editor" target="_blank" tabindex="124">Add a language</a></div>
```

Comment:
The tab index for this link is at 124.

Reply:
Do you mean that it should be at 124 as with the link for `hr` and for `add-language-link`? or should is be 125 and `add-language-link` be 126?

Reply:
I meant it should be above 125, as 124 is Croatian already.

We might as well change it to 129 for the time being, this way we avoid increasing it with each new language.